10-inference-for-regression.Rmd

```diff
@@ -783,23 +783,8 @@ When the conditions for inference for regression are met and the null distributi
 
 
 
-
-
 ## Conclusion
 
-<!--
-
```

Comment:
Are we thinking this is something we won't want to do in v2? I'm fine with that, but it would be something that I think is really missing in intro stats books. It would take a lot of effort to get there I think, but might be something worth exploring later?

Reply:
We can add that back in that case. No harm in keeping it as a `v2 TODO`